### ISSUE TITLE
[Question] padding_side of LlamaTokenizerFast

### ISSUE THREAD

**Issue by x54-729** [huggingface/transformers#25609]
https://github.com/huggingface/transformers/blob/e42587f596181396e1c4b63660abf0c736b10dae/src/transformers/models/llama/tokenization_llama_fast.py#L100

Why `padding_side` is left here? I found out that this may cause different behaviours between `LlamaTokenizer.from_pretrained` and `AutoTokenizer.from_pretrained`.

**Comment by ArthurZucker** [comment]
Hey. We can default to `padding="right"` indeed, not really sure however why/how it can create a different behaviour. Do you have a small reproducer?

**Comment by x54-729** [comment]
> Hey. We can default to `padding="right"` indeed, not really sure however why/how it can create a different behaviour. Do you have a small reproducer?

Not a big problem actually. In some situations I want to test the logits of my model, so `padding_side` may take effect when I want to pad my input. It's not that convenient since I have to set `padding_side` every time.

**Comment by ArthurZucker** [comment]
You can save your tokenizer with `padding_side` and load it from the save folder no?

**Comment by thohag** [comment]
Would be nice to have this inconsistency resolved. It causes issues when working with logits if you are not aware of the special padding. Would be great if all tokenizers were initialized with the same padding_side value.
